docs/docs/reference/crd-reference/app.md

```diff
@@ -165,14 +137,43 @@ spec:
 
 ## Differences between versions
 
-- The `spec.revision` field is introduced in v1alpha2.
-- The pre/post-deployment tasks and evaluations are not
-supported anymore in v1beta1.
+The synopsis of the `KeptnApp` resource
+is changed in `v1beta1` library:
+
+- The `apiVersion` field now references the `v1beta1` version
+  of the `lifecycle.keptn.sh` API.
+- `KeptnApp` is
+  [autogenerated](../../guides/auto-app-discovery.md);
+  you cannot create it manually by applying a manifest.
+- The `app.kubernetes.io/managed-by: keptn` label or annotation
+  is always used for the `v1beta1` library
+  because the `KeptnApp` resource is always autogenerated.
+  In earlier releases, this label was not included in `KeptnApp` resources
+  that were created by applying a manifest
+  that had been created manually.
+- If you are using the context propagation feature,
+  the `name` and `namespace` fields
+  must have identical values to those
+  in the corresponding `KeptnAppContext` resource.
+- The `metadata.version` field is now completely managed by Keptn.
+  You do not increment this value to cause
+  a new execution of tasks and evaluations.
```

Comment:
maybe mention that now the only way to re-execute is to increase the app revision?